### PR TITLE
Bump GeckoView to 68.0.20190506130308

### PR DESCRIPTION
**Pull request by bluemarvin**

### CODE REVIEW

**Comment by philip-lamb** [comment]
@bluemarvin To help me track Geckoview versioning, please note which BZ bug or GH issue this update addresses in a comment on #1068.

**Comment by bluemarvin** [comment]
https://bugzilla.mozilla.org/show_bug.cgi?id=1493225

To fix #682

**Comment by MortimerGoro** [comment]
I can still reproduce the goBack issue:

- Load https://sketchfab.com/
- Back button or home button is unresponsive for a while

File: versions.gradle [hunk 1]
```diff
@@ -24,7 +24,7 @@ ext.deps = [:]
 def versions = [:]
 // GeckoView versions can be found here:
 // https://maven.mozilla.org/?prefix=maven2/org/mozilla/geckoview/
-versions.gecko_view = "68.0.20190422094240"
+versions.gecko_view = "68.0.20190506130308"
```
Comment (cvan) [inline versions.gradle]: since it's been a few days, I bumped the version: `68.0.20190508111321`. seems to work well so far.

**Comment by bluemarvin** [comment]
We are going to need a newer version of GeckoView to pick up the CrashHandler fix so I'll close this and open a new PR once the fix is in GV nightly.

**Pull request closed without merging**